projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
```

Comment:
Since you're using a TypeScript file and using TypeScript types, I think we can remove the PropTypes from this

Reply:
Ahh yes, thanks!